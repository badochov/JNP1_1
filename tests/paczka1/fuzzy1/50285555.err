Error in line 14: kvi18 A876 36,5
Error in line 27: 7Mrl2 S149 50,6
Error in line 16: rOP S149 93,6
