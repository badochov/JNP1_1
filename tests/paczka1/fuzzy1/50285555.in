?
4pKzumWOW S149 2,9
A760 A876 14,7
YKDZwE3mF A876 77,8
YKDZwE3mF A876 119,8
rOP A760 88,4
kvi18 A876 21,5
YKDZwE3mF A876 62,7
4pKzumWOW S149 0,2
7Mrl2 S149 97,6
A760 A876 6,3
neki7jOMX A876 81,9
kvi18 A876 35,4
kvi18 A876 36,5
rOP A760 99,5
rOP S149 93,6
4pKzumWOW S149 97,5
?S149
4pKzumWOW S149 210,6
?neki7jOMX
?
A760 A760 50,4
kvi18 S149 38,1
A760 A760 38,3
7Mrl2 S149 40,5
?
7Mrl2 S149 50,6
YKDZwE3mF A876 72,7
A760 S149 50,9
7Mrl2 A585 66,3
7Mrl2 A585 27,2
neki7jOMX A876 249,8
?YKDZwE3mF
rOP A585 98,6

4pKzumWOW A760 43,2
4pKzumWOW A760 199,6
YKDZwE3mF S149 35,3
7Mrl2 A876 16,1
7Mrl2 A876 2,6