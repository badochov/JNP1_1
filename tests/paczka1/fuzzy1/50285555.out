S149 2,7
4pKzumWOW S 115,8
A760 A 8,4
YKDZwE3mF A 42,0
kvi18 A 13,9
rOP A 11,1
S149 115,8
A760 11,1
A876 64,3
4pKzumWOW S 115,8
7Mrl2 S 57,1
A760 A 20,5
YKDZwE3mF A 42,0
kvi18 A 13,9
rOP A 11,1
S149 172,9
A760 23,2
A876 64,3
YKDZwE3mF A 52,0
